corehq/apps/app_manager/tests/test_suite_remote_request.py

```diff
@@ -788,3 +788,54 @@ def test_exclude_from_search(self, *args):
         </partial>
         """
         self.assertXmlPartialEqual(expected, suite, "./remote-request[1]/session/query/prompt[@key='name']")
+
+    def test_case_search_title_translation(self, *args):
```

Comment:
Nit: I'd split this into two tests and name them so it's clear that one is testing legacy behavior and the other is testing the new behavior.